Formula/nethack.rb

```diff
@@ -22,6 +22,7 @@ class Nethack < Formula
   def install
     # Build everything in-order
     ENV.deparallelize
+    ENV["HOMEBREW_OPTIMIZATION_LEVEL"] = "O0"
```

Comment:
The preferred API for this is to simply call `ENV.O0`. Please also write a comment explaining the need for a different optimization level.

Reply:
Done. Thanks!